src/solve.jl

```diff
@@ -32,60 +35,92 @@ function init(prob::AbstractDDEProblem{uType,tType,lType,isinplace}, alg::algTyp
     end
 
     if typeof(alg.alg) <: OrdinaryDiffEqCompositeAlgorithm
-        interp_data = OrdinaryDiffEq.CompositeInterpolationData(integrator.sol.interp,
-                                                                interp_f)
+        ode_interp_data = OrdinaryDiffEq.CompositeInterpolationData(integrator.sol.interp,
```

Comment:
this is making it hard to review. Is half of what's going on an unrelated renaming?